scripts/ci/post_style_errors_on_github.sh

```diff
@@ -4,16 +4,12 @@ ERROR_LOG="$1"
 
 function report_error_to_github
 {
-    if [ -z $CIRCLE_PR_NUMBER ]
+    if [[ $CIRCLE_PR_NUMBER != "" ]]
     then
         CIRCLE_PR_NUMBER="${CIRCLE_PULL_REQUEST//[^0-9]/}"
     fi
 
-    ERROR_MSG=$(cat $ERROR_LOG)
-
-    echo $ERROR_MSG
-
-    if [ ! -z $CI ]
+    if [[ $CI == "true" ]]
```

Comment:
Could also be `$CI != ""` but I see in CircleCI that this is set to `true` and I think this check is more readable.